spec/spec_helper_acceptance.rb

```diff
@@ -116,3 +105,4 @@
     apply_manifest(pp, catch_changes: true)
   end
 end
+>>>>>>> acceptance-copy
```

Comment:
@ekohl is this supposed to be here, or from conflict resolution?

Reply:
So I didn't properly restore it. I'll have to look into this.